### ISSUE TITLE
Empty link causing accessibility issue

### ISSUE THREAD

**Issue by susiestrath** [PlaneteStream/moodle-atto_planetestream#8]
**Affected Version: 6.541 (2023033100)** (Unable to check the latest version)

Whilst running accessibility tools on our Moodle service, we realised that this error is consistently reported across all pages with eStream embeds:

![CleanShot 2024-04-04 at 15 36 44](https://github.com/PlaneteStream/moodle-atto_planetestream/assets/79458670/8a0102be-5836-43dc-b974-f153e270cdbf)

The link is placed in the tree immediately after the iframe:
![CleanShot 2024-04-04 at 15 39 52](https://github.com/PlaneteStream/moodle-atto_planetestream/assets/79458670/326404c4-1b4d-40cc-ba30-e6557bffbcf9)

Whilst we understand that the link is not visible to users, it is considered a failure of WCAG Level A:

- [SC 2.4.4: Link Purpose (In Context) (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)
- [SC 4.1.2: Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

This issue impacts users navigating using a keyboard and it will also be announced to screen readers. If accidentally clicked, the link takes to a 404 page, which may confuse users even more.

Using an a tag in this case may not be appropriate. If the a tag is the only option but is just existing for the purpose of rendering the iframe, this should be at least hidden or removed from the tree and should not be keyboard focusable.

I hope this makes sense, please let me know if you need further information.

Susie

**Comment by PlaneteStream** [comment]
Hi Susie,

Thanks for bringing this to our attention. I've just updated the YUI folder and version file with some changes to remove the links as I believe they're a legacy component that's no longer necessary. Retested and working on our end, will do the same for the TinyMCE / Tiny plugins now and updated the Githubs and Moodle plugin repos soon.

It's worth noting that this won't remove any existing links on pages as Moodle doesn't generate these dynamically - the code is already there so would need manually removing and replacing.

**Comment by susiestrath** [comment]
Thank you for fixing that so quickly, that is great news! Much appreciated :)
